casper/src/test/scala/coop/rchain/casper/MultiParentCasperMergeSpec.scala

```diff
@@ -213,16 +217,16 @@ class MultiParentCasperMergeSpec
         "!C",
         "4X",
         "4!",
-        "4!!",
+        //"4!!",
         "PX",
         "P!",
         "P!!",
-        "!!X",
-        "!!4",
-        "!!C",
+        //"!!X",
+        //"!!4",
+        //"!!C",
```

Comment:
what's up with this?

Reply:
Ah, I see - they were excluded by the filter :+1:

Reply:
I would add those tests in a separate PR as these are not directly connected with peek (and hence this PR)

Reply:
I think we should also consider making our tests for merge rules smaller. Maybe it'd make sense to do some of:

1) remove the notion of volatile events from tests, as they're secondary to the deploy-boundary-agnostic, original merge rules

2) add test cases for !P and !!P (not covered by the spreadsheet)

The two above would probably reduce the number of `MergeabilityCase`-s to 4